app/build.gradle

```diff
@@ -95,7 +95,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 
     // Hover SDK
-    def sdk_version = "1.7.2-stax-1.1.4-pro"
+    def sdk_version = "1.7.2-stax-1.1.5-pro"
     releaseImplementation "com.hover:android-sdk:$sdk_version"
     debugImplementation project(":hover.sdk")
     debugImplementation 'com.android.volley:volley:1.1.1'
```

Comment:
Retrofit would be be best. Offers great features out of the box - Flows, Suspending Calls and this works well with Room DB setup on the app